autoscaler/main.go

```diff
@@ -164,6 +167,22 @@ func main() {
 		os.Exit(1)
 	}
 
+	ctx := ctrl.SetupSignalHandler()
+	gatewayAutoscaler := collectormetrics.NewAutoscaler(mgr.GetClient(),
+		collectormetrics.WithCollectorsGroup(odigosv1.CollectorsGroupRoleClusterGateway),
+		collectormetrics.WithInterval(15*time.Second),
```

Comment:
The 15 seconds interval should be a const or probably something we to be able to set in a config map, right?